cmake/Modules/dotnet/Directory.Build.props.in

```diff
@@ -0,0 +1,5 @@
+<Project>
```

Comment:
The directory for building each of the projects must be independent. The best option I have found makes it depends on his project name.

Reply:
Ok, but why does it need to be independent? What's the rationale?

Reply:
Ok. If my memory doesn't fail me, xunit required a different target framework. As buildings were made in the same directory, concurrent buildings that required different target frameworks failed. 

This change lets to build different project in different build directories to avoid this problem.